src/Ilios/AuthenticationBundle/Voter/TemporaryFileSystemVoter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Ilios\AuthenticationBundle\Voter;
+
+use Ilios\CoreBundle\Classes\TemporaryFileSystem;
+use Ilios\CoreBundle\Entity\UserInterface;
+
+/**
+ * Class TemporaryFileSystemVoter
+ * @package Ilios\AuthenticationBundle\Voter
+ */
+class TemporaryFileSystemVoter extends AbstractVoter
```

Comment:
NICE!  Love using the voter in this way to authorize things besides model access.